circle.yml

```diff
@@ -16,7 +16,6 @@ dependencies:
     - sudo service apache2 restart
     - chmod a+rwx backup log public/rss upload
     - sudo composer self-update --no-interaction
-    - composer config --global --no-interaction github-oauth.github.com $GITHUB_OAUTH_TOKEN
```

Comment:
@jamiesnape Any clue what this line does?  Removing it fixes CircleCI testing for forks of CDash.  Our other option is apparently to check the value of this token into our repo.
https://circleci.com/docs/fork-pr-builds